### PR TITLE
Updating phpstorm meta files with magerun 2.1.0

### PR DESCRIPTION
**Pull request by boesbo**
Updating phpstorm meta files with magerun 2.1.0. This solves the compatibility problem with the PHPStorm Stubs library used for example in the vscode PHP-Intelephense extension.

### Related Pull Requests
https://github.com/netz98/n98-magerun/pull/1141

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Review by colinmollenhour — COMMENTED** [review]
It looks like this adds some classes that don't exist in the core.

**Review by Flyingmana — DISMISSED** [review]

**Review by kiatng — APPROVED** [review]

File: .phpstorm.meta.php/magento_blocks_methods.meta.php [hunk 1]
```diff
@@ -280,6 +280,7 @@
             'adminhtml/notification_inbox' => \Mage_Adminhtml_Block_Notification_Inbox::class,
             'adminhtml/notification_security' => \Mage_Adminhtml_Block_Notification_Security::class,
             'adminhtml/notification_survey' => \Mage_Adminhtml_Block_Notification_Survey::class,
+            'adminhtml/notification_symlink' => \Mage_Adminhtml_Block_Notification_Symlink::class,
```
Comment (sreichel) [inline .phpstorm.meta.php/magento_blocks_methods.meta.php:283]: No such file.

File: .phpstorm.meta.php/magento_blocks_methods.meta.php [hunk 2]
```diff
@@ -572,6 +573,7 @@
             'adminhtml/sales_order_view_tab_history' => \Mage_Adminhtml_Block_Sales_Order_View_Tab_History::class,
             'adminhtml/sales_order_view_tab_info' => \Mage_Adminhtml_Block_Sales_Order_View_Tab_Info::class,
             'adminhtml/sales_order_view_tab_invoices' => \Mage_Adminhtml_Block_Sales_Order_View_Tab_Invoices::class,
+            'adminhtml/sales_order_view_tab_prova' => \Mage_Adminhtml_Block_Sales_Order_View_Tab_Prova::class,
```
Comment (sreichel) [inline .phpstorm.meta.php/magento_blocks_methods.meta.php:576]: No such file.

**Review by sreichel — CHANGES_REQUESTED** [review]
Please remove that two classes.

**Review by sreichel — APPROVED** [review]

**Review by kiatng — APPROVED** [review]

**Review by colinmollenhour — APPROVED** [review]

**Pull request merged**